Makefile

```diff
@@ -199,8 +206,15 @@ ifeq ($(BOARD), qemu_x86)
 else
 ifeq ($(DEV), ashell)
 	@cat fragments/prj.conf.ashell >> prj.conf
+	@cat fragments/prj.mdef.ashell >> prj.mdef
+ifeq ($(filter ide,$(MAKECMDGOALS)),ide)
+	@echo CONFIG_USB_CDC_ACM=n >> prj.conf
+else
+	@echo CONFIG_USB_CDC_ACM=y >> prj.conf
+endif
+else
+	@cat fragments/prj.conf.base >> prj.conf
```

Comment:
They do, but they get it from a few lines above this.  However this does point out that this else isn't necessary, I'll remove it.  Thanks.